base/system/diskpart/diskpart_msg.mc

```diff
@@ -6699,22 +6716,23 @@ Example:
     SELECT VOLUME=C:\\MountH
 .
 Language=Romanian
-    Selects the specified volume and shifts the focus to it.
+    Selectează volumul specificat și mută focalizarea pe el.
 
-Syntax:  SELECT VOLUME={<N> | <D>}
+Sintaxă:  SELECT VOLUME={<N> | <D>}
 
-    VOLUME=<N>  The number of the volume to receive the focus.
+    VOLUME=<N>  Numărul volumului care urmează să primească marcaj.
```

Comment:
In other instances, I noticed you've chose to translate the same "focus" as "focalizare", so please take care of consistency.